src/main.rs

```diff
@@ -31,22 +27,38 @@ fn content(content_type: &str, reply: impl Reply) -> impl Reply {
 
 // #[get("/")]
 fn index() -> impl Reply {
-    content("text/html; charset=utf-8", INDEX_HTML)
+    #[cfg(not(debug_assertions))]
+    static index_html: &str = include_str!("../frontend/static/index.html");
```

Comment:
these var names give a warning in release mode, I just realised. I could either suppress them on this line or add extra compiler macros to reference different vars in the `content()` call. What do you think @charliesome ?

Reply:
If keeping different var names, could push these statics back up to top of file, but I thought having them close to the dynamic version would be useful to keep paths in sync.

Reply:
You can just use a let binding here - the string won't be copied

Reply:
ah true. nice.